integrations/docker/images/chip-build-tizen/secret-tool.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+# Dummy Password Manager for Tizen Studio CLI
+
+import fcntl
+import os
+import pickle
+import sys
+from argparse import ArgumentParser
+
+
+class Secrets:
+
+    def __init__(self, filename: str):
+        self.filename = filename
+        self.dirty = False
+        self.secrets = {}
+        self.fp = None
+
+    def __enter__(self):
+        self.fp = open(self.filename, "a+b")
+        fcntl.flock(self.fp.fileno(), fcntl.LOCK_EX)
+        self._load()
+        return self
+
+    def __exit__(self, _type, value, tb):
+        if self.dirty:
+            self._save()
+        fcntl.flock(self.fp.fileno(), fcntl.LOCK_UN)
+        self.fp.close()
+
+    @staticmethod
+    def _build_key(label: str, **kw):
+        return label + ":" + str(tuple(sorted(kw.items())))
+
+    def _load(self):
+        try:
+            self.fp.seek(0)
+            self.secrets = pickle.load(self.fp)
+        except EOFError:
+            # Unpickling an empty file is not an error for us
+            pass
+        except ValueError as e:
+            print("ERROR: " + str(e), file=sys.stderr)
+
+    def _save(self):
+        try:
+            self.fp.seek(0)
+            self.fp.truncate()
+            pickle.dump(self.secrets, self.fp)
+        except IOError as e:
+            print("ERROR: " + str(e), file=sys.stderr)
+
+    def clear(self, label: str, **kw):
+        key = self._build_key(label, **kw)
+        self.secrets.pop(key, None)
+        self.dirty = True
+
+    def store(self, label: str, password: str, **kw):
+        key = self._build_key(label, **kw)
+        self.secrets[key] = password
+        self.dirty = True
+
+    def lookup(self, label: str, **kw):
+        key = self._build_key(label, **kw)
+        return self.secrets.get(key, "")
+
+
+parser = ArgumentParser()
```

Comment:
Minor- adding "help" strings for the command line options would go a long way.

Reply:
Done.